### PR TITLE
Change mixin approach

### PR DESCRIPTION
**Pull request by CoMPaTech**

### CODE REVIEW

**Comment by CoMPaTech** [comment]
Do not just merge, we should re-evaluate why the test for the slave-boiler has a different outcome

File: custom_components/plugwise/models.py [hunk 1]
```diff
@@ -407,46 +391,43 @@ class PlugwiseBinarySensorEntityDescription(
...
     PlugwiseSensorEntityDescription(
         key="maximum_boiler_temperature",
         translation_key="maximum_boiler_temperature",
         device_class=SensorDeviceClass.TEMPERATURE,
         entity_category=EntityCategory.DIAGNOSTIC,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
-        value_fn=lambda data: data["maximum_boiler_temperature"]["setpoint"],  # type: ignore [index]
+        pw_key="setpoint",
+        pw_lookup="maximum_boiler_temperature",
     ),
```
Comment (bouwew) [inline custom_components/plugwise/models.py:431]: I think we should remove this one.
This one will always be a select, never a sensor.

Comment (bouwew) [reply custom_components/plugwise/models.py:431]: The rest looks very good!

Comment (bouwew) [reply custom_components/plugwise/models.py:431]: Should this construct be used for `number` and `select` as well?

Comment (CoMPaTech) [reply custom_components/plugwise/models.py:431]: If we don't need the lookup for 'other than sensors' dict we can remove part of the lookup function. I was tempted to do select as well but stopped in time to first make sure it was Mypy-compatible (which took another hour before able to commit)

**Comment by CoMPaTech** [comment]
As discussed, number needs more thinking cap before processing/simply 4x-multiplying the code. Select included as per latest commit.

**Comment by CoMPaTech** [comment]
Draft as per 'cast'-ing and awaiting Core feedback

**Pull request closed without merging**